# HG changeset patch
# User Mike Miller <mtmiller@octave.org>
# Date 1458358432 25200
#      Fri Mar 18 20:33:52 2016 -0700
# Node ID 82089c8ed7fadbe75e25cbf68a713196d14f6908
# Parent  8e9833b8791d7628192174659a076f6bcddfa716
textscan.h: Delete unused member function textscan::get_field

* textscan.h (textscan::get_field): Delete unimplemented member function.

diff --git a/libinterp/corefcn/textscan.h b/libinterp/corefcn/textscan.h
--- a/libinterp/corefcn/textscan.h
+++ b/libinterp/corefcn/textscan.h
@@ -161,18 +161,16 @@ private:
 
   // Helper methods.
   std::string read_until (delimited_stream& is, const Cell& delimiters,
                           const std::string& ends) const;
 
   int lookahead (delimited_stream& is, const Cell& targets, int max_len,
                  bool case_sensitive = true) const;
 
-  char *get_field (delimited_stream& isp, unsigned int width) const;
-
   bool match_literal (delimited_stream& isp, const textscan_format_elt& elem);
 
   int  skip_whitespace (delimited_stream& is, bool EOLstop = false);
 
   int  skip_delim (delimited_stream& is);
 
   bool is_delim (unsigned char ch) const
   {
